src/pages/index.story.tsx

```diff
@@ -0,0 +1,71 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { ReplaySubject } from "rxjs"
+import { Analytics } from "../analytics"
+import { TranslationContext } from "../components/contexts"
+import { withAutoUnsubscribe } from "../components/with-auto-unsubscribe"
+import { SocketPayload } from "../components/with-socket"
+import { defaultTranslations } from "../i18n/default-translations"
+import { createTranslate } from "../i18n/translate"
+import { views } from "../views"
+import { _Controller } from "./index"
+
+const analyticsStub: Analytics = {
+  sendCustomEvent: () => undefined,
+  sendPageView: () => undefined,
+}
+
+const socket = new ReplaySubject<SocketPayload>()
+const Controller = withAutoUnsubscribe(_Controller)
+
+const View = () => {
+  return (
+    <div style={{ width: "320px", height: "568px", position: "relative" }}>
+      <TranslationContext.Provider value={createTranslate(defaultTranslations)}>
+        <Controller analytics={analyticsStub} socket={socket} />
+      </TranslationContext.Provider>
+    </div>
+  )
+}
+
+storiesOf("Controller", module)
+  .add("Start", () => {
+    socket.next({
+      event: "state",
+      payload: { activeView: views.CONTROLLER_START },
+    })
+
+    return View()
```

Comment:
since this is a stateless functional component we should invoke it by `<View/>`

Reply:
Will change